test/command/test-logical-range-filter.rb

```diff
@@ -34,6 +34,7 @@ def test_ordered_arguments
       limit          = "20"
       filter         = "value == 10"
       output_columns = "_key, timestamp"
+      sort_keys      = "-_score"
```

Comment:
`_score` isn't valid `sort_keys` for `logical_range_filter` because it's not the same order as shard key.